src/meta/MetaServiceUtils.cpp

```diff
@@ -302,5 +311,43 @@ cpp2::ErrorCode MetaServiceUtils::alterColumnDefs(std::vector<nebula::cpp2::Colu
     return cpp2::ErrorCode::E_NOT_FOUND;
 }
 
+cpp2::ErrorCode MetaServiceUtils::alterSchemaProp(std::vector<nebula::cpp2::ColumnDef>& cols,
+                                                  nebula::cpp2::SchemaProp& prop,
+                                                  cpp2::AlterSchemaProp alterProp) {
+    auto propType = alterProp.get_type();
+    std::string value = alterProp.get_value();
+    switch (propType) {
+        case cpp2::AlterSchemaPropType::TTL_DURATION:
+            // Later check ttl_duration value, <=0 to =0
+            prop.set_ttl_duration(folly::to<int>(value));
+            return cpp2::ErrorCode::SUCCEEDED;;
+        case cpp2::AlterSchemaPropType::TTL_COL:
+            for (auto& col : cols) {
+                if (col.get_name() == value) {
+                    prop.set_ttl_col(value);
+                    return cpp2::ErrorCode::SUCCEEDED;
+                }
+            }
+            LOG(WARNING) << "TTL Column not found : " << value;
+            return cpp2::ErrorCode::E_NOT_FOUND;
+        default:
+            return cpp2::ErrorCode::E_UNKNOWN;
+    }
+}
+
+Status MetaServiceUtils::checkSchemaTTLProp(nebula::cpp2::SchemaProp& prop) {
```

Comment:
I think it makes more sense to put this check on graphd

Reply:
Good,  👍
I will fixed

Reply:
I carefully thought about it.
Here  `checkSchemaTTLProp` uses `alter tag/edge` sentence. 
`checkSchemaTTLProp` can be executed after the schema of the alter statement is updated in meta.